cmd/clusterctl/client/config/providers_client.go

```diff
@@ -21,7 +21,6 @@ import (
 	"os"
 	"sort"
 	"strings"
-
```

Comment:
Please fix the formatting findings reported by golagnci-lint.

(probably most of them should be fixed after running `make lint-fix`)

Reply:
Thank you, it is done